e2e-polybft/e2e/consensus_test.go

```diff
@@ -748,3 +749,80 @@ func TestE2E_Consensus_ChangeVotingPowerByStakingPendingRewards(t *testing.T) {
 		t.Errorf("voting power did not change in consensus for %d epochs", numOfEpochsToCheckChange)
 	}
 }
+
+func TestE2E_Deploy_Dummy_Contracts(t *testing.T) {
+	var newValue = big.NewInt(234586)
```

Comment:
Nitpick (less to type in and more consistent with the rest of the codebase):

```suggestion
	newValue := big.NewInt(234586)
```